src/routes/executions.js

```diff
@@ -0,0 +1,46 @@
+import { Router } from "express"
+import { Container } from "typedi"
+import { Auth, ExecutionService } from "../services"
+import config from "../config"
+
+export default appRouter => {
+  const router = Router()
+  const authService = Container.get(Auth)
+
+  //Validação do Codigo de Entrada
+  router.post("/", authService.middlewares.requireAppAdministration, (req, res) => {
+    const executionService = Container.get(ExecutionService)
+    const deviceToken = req.body.deviceToken
+
+  // Confere se o Token for vazio e se não tiver nenhuma execução
+    if (deviceToken == null || req.body.executions.length <= 0) {
+      res.status(400).json({
+        "code": "bad_request/missing_fields",
+        "message": "fields 'deviceToken' and 'executions' are required, and there should be at least one execution"
+      })
+      return
+    }
+
+    for (var item of req.body.executions) {
```

Comment:
Warning no linter, trocar `var` por `const`:

```suggestion
    for (const item of req.body.executions) {
```